src/components/Banner/MucBanner.vue

```diff
@@ -73,13 +91,22 @@ const typeAriaLabel = computed(() => {
       >
         <div class="container-fluid">
           <svg class="icon">
-            <use href="#icon-information" />
+            <use :href="typeIcon" />
           </svg>
-          <p>
-            <slot />
-          </p>
+          <slot />
         </div>
       </div>
     </div>
   </div>
 </template>
+
+<style scoped>
+.m-banner--success {
+  background-color: #f1f6f3;
+  border-bottom: 1px solid #3a7f53;
+}
+
+.m-banner--success .icon {
+  color: #3a7f53;
```

Comment:
All other icons to not have a color - therefore success shouldn't either.
![grafik](https://github.com/it-at-m/muc-patternlab-vue/assets/38280183/c1059801-8889-4fec-b9c2-90bdfe737069)

Reply:
I would say you are right, but: the other icons get their color from patternlab and thus I want to follow it.

The new success-variant isn't yet available in patternlab but was designed specifically this way by our webmanagement team. Thus I chose to accept this difference.

Reply:
The same scenario was on the callout component - the success variant did not exist (yet).
We did not set a color there too ... but this does look different ![grafik](https://github.com/it-at-m/muc-patternlab-vue/assets/38280183/fd038d39-f9d8-4880-909a-78fba76a0345)